net-wireless/blueberry/blueberry-1.3.4.ebuild

```diff
@@ -21,6 +21,7 @@ RDEPEND="${PYTHON_DEPS}
 	dev-python/dbus-python[${PYTHON_USEDEP}]
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	dev-python/setproctitle[${PYTHON_USEDEP}]
+	dev-python/xapp[${PYTHON_USEDEP}]
```

Comment:
Could you look into enabling python3.7 with xapp?

Reply:
I'm not the maintainer of this package but it should not be hard. The version in the tree is quite old and could be updated. But I guess I can't do that here without opening a new bug?

Reply:
just add it here please, its maintained by @gentoo/cinnamon which looks pretty dead. I'll test and make sure it won't break anything.

Reply:
Rationale being it feels foolish **dropping** python support with new version bumps, the reverse deps should be investigated and updated first. If possible, it can really be that xapp doesn't support python3.7 at all, which in case means that long-term it either needs to be patched or blueberry needs to be patched to not depend on xapp.

Reply:
I added xapp with python3.7 enabled, it should work just fine, as for some other distros.
And I re-enabled python3.7 for blueberry